app/views/custom/welcome/_header.html.erb

```diff
@@ -0,0 +1,22 @@
+<% if header.present? %>
+  <div class="jumbo highlight">
+    <div class="row">
+      <div class="small-12 medium-6 column <%= "medium-9 small-centered text-center" unless header.image.present? %>">
+        <span><%= header.label %></span>
+        <h1><%= header.title %></h1>
+        <p class="lead"><%= header.description %></p>
+        <div class="small-12 medium-6 large-4 <%= "small-centered" unless header.image.present? %>">
+          <%= link_to header.link_text, header.link_url, class: "button expanded large" %>
+        </div>
+      </div>
+
+      <% if header.image.present? %>
+        <div class="small-12 medium-6 column">
+          <%= image_tag(header.image_url(:large),
+                        class: "margin",
+                        alt: header.image.title) %>
+        </div>
+      <% end %>
```

Comment:
Instead of copying the view to remove the `background-header` div, do we get the same result adding a `display: none` CSS rule to this element?